### ISSUE TITLE
The parseFields substitution doesn't correctly work with conditions

### ISSUE THREAD

**Issue by ghpxi** [statgen/locuszoom#160]
From this method:
https://github.com/statgen/locuszoom/blob/develop/dist/locuszoom.app.js#L398

Documentation says that condition format should be like this:
`{{#if {{field_name}} }} Conditional text {{/if}}`
But regexp match only this format:
`{{#if field_name}} Conditional text {{/if}}`
It works correcly, but you should either update documentation or fix regex

You can see the tests here:
https://regex101.com/r/bU8J04/2

**Comment by abought** [comment]
Thanks for catching this!

The short answer is that the second version is the correct and [intended](https://github.com/statgen/locuszoom/blob/develop/test/unit/LocusZoom.js#L211) grammar; I've updated the source code docs accordingly in f096851.

That said, one place where curly braces are still valid is when using namespaces in raw layouts, eg:
[`{{#if {{namespace[catalog]}}rsid}}`](https://github.com/statgen/locuszoom/blob/develop/assets/js/app/Layouts.js#L386). The reason is that `{{namespace[datasource_name]}}` will get resolved when the layout is first retrieved, so that what parseFields eventually sees is `{{# if catalog:rsid }}`.

**Comment by ghpxi** [comment]
Thank you for the quick answer.
By the way, the regex doesn't accept space after a variable. You can check it [here](https://regex101.com/r/bU8J04/2). This small thing is confusing. So, the documentation is still invalid.

The place where we use LocusZoom.js isn't public, so I cannot share it, at least for now.

**Comment by abought** [comment]
Thanks for the feedback!

Indeed, that was fixed in a followup commit. (let me know if it's still in error). 
https://github.com/statgen/locuszoom/blob/develop/assets/js/app/LocusZoom.js#L360

I've also added the documentation in a more officially visible place on our wiki; it seems this was a hidden feature:
https://github.com/statgen/locuszoom/wiki/Layouts#conditional-expressions

If your tool does become public of course let us know; we're happy to get feedback from any channel.

We're thinking of sending out a user survey at some point to find out how people use LZ.js (and what sorts of improvements would help them for common use cases)... if you're on our google group mailing list, we would definitely announce it there.
